### ISSUE TITLE
Catch infinite loops before they crash editor 

### ISSUE THREAD

**Issue by catarak** [processing/p5.js-web-editor#77]
If there are infinite loops, the whole editor will crash. The editor should check for this before inserting the code into the iframe.

**Comment by lmccart** [comment]
https://github.com/jsbin/loop-protect might be helpful?

**Comment by kshivanku** [comment]
Hi,

I faced a similar issue. I was just practicing while loop in the editor. Every time I started a loop the editor would hang and I had to refresh the page. It took me a couple of attempts to realize that I was in the "play" mode and the browser was probably hanging because it got stuck in an infinite loop.

Screenshot attached.
<img width="1280" alt="editor" src="https://cloud.githubusercontent.com/assets/12475379/18422224/d560919a-7865-11e6-80f3-4d5c36c8d16c.png">

This can be quite disastrous as there is no autosave yet and a user can easily forget which mode he is in while working. Just wanted to bring this to your notice.
